server/middlewares/validate.js

```diff
@@ -1,39 +1,128 @@
 import Validator from 'validatorjs';
+
 /**
  *
  *
  * @class Validate
  */
 class Validate {
   /**
-   *
-   * @param {request} request
-   *
-   * @param {response} response
-   *
-   * @param {function} next
-   *
-   * @returns {Object} - JSON object and status code
-   *
-   * @memberof Validate
-   */
-  static entryId(request, response, next) {
```

Comment:
Seems you have removed this but you still require/use it.
You could always have both
`static userId` and `static entryId`

This is why your test is failing by the way